module/attribute/src/Persistence/Dbal/Projector/Attribute/AbstractAttributeParameterChangeEventProjector.php

```diff
@@ -41,11 +41,11 @@ public function __construct(Connection $connection, SerializerInterface $seriali
     }
 
     /**
-     * @param AttributeParameterChangeEvent $event
+     * @param DomainEventInterface $event
      *
      * @throws DBALException
      */
-    public function __invoke(AttributeParameterChangeEvent $event): void
+    protected function project(DomainEventInterface $event): void
```

Comment:
This is not good solution.
Interface DomainEventInterface has no methods: 
- getTo()
- getName(), 

With are used in project method body.